content/backmatter.tex

```diff
@@ -672,98 +705,109 @@ \section{Version 1.5}
 \item Added team-based communication-management functions:
   \FUNC{shmem\_team\_create\_ctx} and
```

Comment:
```suggestion
  \FUNC{shmem\_team\_create\_ctx} and \\
```

Reply:
Do these `\FUNC` bleed into the margin? If so, the same also happens with "Added support for a multiple-element point-to-point synchronization API with the functions". I would prefer to not adjust the line breaks on this one.

Reply:
I will double check before merging.

Reply:
Adding the line break here didn't change what landed on each line, but it fixed some bad whitespace that LaTeX had inserted to fully justify the line containing `shmem_team_create_ctx`.

<img width="917" alt="Screen Shot 2020-04-28 at 4 10 11 PM" src="https://user-images.githubusercontent.com/3397618/80532928-d9182b80-896a-11ea-9862-94cdc3bd5feb.png">